build.gradle

```diff
@@ -866,3 +866,8 @@ project(":samza-test_$scalaSuffix") {
     from(configurations.archives.artifacts.files) { into("lib/") }
   }
 }
+
+// SAMZA-2473 read wrapper version from gradle.properties
+wrapper {
+  gradleVersion = project.gradleVersion
```

Comment:
Could you please add a comment in `gradle.properties` right above `gradleVersion` to re-run `./gradlew wrapper` and commit the generated changes when upgrading Gradle in the future?

Reply:
Great suggestion; will do.